sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -1362,7 +1362,7 @@ def test_queue_by_queue_client_conn_str_receive_handler_with_autolockrenew(self,
                             receiver.complete_message(message)
                             raise AssertionError("Didn't raise MessageLockLostError")
                         except MessageLockLostError as e:
-                            assert isinstance(e.inner_exception, AutoLockRenewTimeout)
```

Comment:
In the previous MessageLockLostError that we raised from ServiceBusReceiver() we set the inner_exception as AutoLockRenewTimeout. I am not sure why this was the case, the inner_exception now is the MessageException raised when we get a Disposition error from the service.

Reply:
Technically breaking since we are changing the type and if someone else was relying on it for logic, but its also the `inner_exception` here. ALR doesn't raise an error when its gone past the lock time?